lib/elixir/lib/code.ex

```diff
@@ -659,6 +663,33 @@ defmodule Code do
   The opposite of converting a string to its quoted form is
   `Macro.to_string/2`, which converts a quoted form to a string/binary
   representation.
+
+  ## The `:static_atom_encoder` function
+
+  When `static_atom_encoder: &my_encoder/2` is passed as an argument,
+  `my_encoder/2` is called every time the tokenizer needs to create a
+  'static' atom. Static atoms are atoms in the AST that function as
```

Comment:
Nitpick, but we don't use single quotes `'` anywhere in the Elixir docs that I know of.

```suggestion
  "static" atom. Static atoms are atoms in the AST that function as
```